source/utilities.tex

```diff
@@ -815,7 +815,7 @@
 \throws Nothing.
 \end{itemdescr}
 
-\xref ISO C~7.22.1.3, ISO C~7.22.1.4.
+\xref ISO C 7.22.1.3, ISO C 7.22.1.4
```

Comment:
Elsewhere we don't repeat ISO C for each reference if there is more than one. (See [c.mb.wcs], [c.math.abs], [c.math.fpclass].) Suggest removing the second ISO C here.